code/modules/surgery/organs/organ_internal.dm

```diff
@@ -257,8 +257,20 @@
 	attack_verb = list("licked", "slobbered", "slapped", "frenched", "tongued")
 
 /obj/item/organ/internal/tongue/proc/TongueSpeech(var/message)
+	spawn(0)
+		speakDevilName(message)
 	return message
 
+/obj/item/organ/internal/tongue/proc/speakDevilName(var/message)
```

Comment:
Care to explain why?  Unless there is a devil, this proc will simply see that the list is empty and do nothing.  Even if there's three devils, it will only be about as intensive as the lizardman speech replacement.